src/pages/aclmanagement.html

```diff
@@ -24,16 +24,34 @@ <h1 id="acl-management-view-title">
             </div>
             <ul class="nav nav-tabs acl-tabs">
                 <li class="resource-tab nav-item">
```

Comment:
Remove these classes from the `li` tags. They are invalid for this view.

Reply:
done